client/transactions/declarations.d.ts

```diff
@@ -38,6 +38,7 @@ declare module '@woocommerce/components' {
 		path?: string;
 		query?: Query;
 		showDatePicker: boolean;
+		onAdvancedFilterAction?: ( args: any ) => void;
```

Comment:
You've used `any` here, but from your actual usage of this callback it seems it should be `string`. Do we know if it will always be a string value?

Reply:
yes, it's always string. thx.

Reply:
fixed in https://github.com/Automattic/woocommerce-payments/pull/8020/commits/7fc466f769175892cd051bfc3f3a0802801932ba . thank you